db/migrate/20170202195228_migrate_ansible_tower_configuration_manager_settings_to_automation_manager.rb

```diff
@@ -0,0 +1,45 @@
+class MigrateAnsibleTowerConfigurationManagerSettingsToAutomationManager < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    serialize :value
+  end
+
+  def up
+    say_with_time('Migrate /ems_refresh/ansible_tower_configuration% to be of Automation Manager') do
+      SettingsChange.where('key LIKE ?', '/ems_refresh/ansible_tower_configuration%').each do |s|
+        s.key = s.key.sub(%r{^/ems_refresh/ansible_tower_configuration}, '/ems_refresh/ansible_tower_automation')
+        s.save!
+      end
+    end
+
+    say_with_time('Migrate /workers/worker_base/queue_worker_base/ems_refresh_worker/ems_refresh_worker_ansible_tower_configuration%' \
+      ' to be of Automation Manager') do
+      SettingsChange.where('key LIKE ?', '/workers/worker_base/queue_worker_base/ems_refresh_worker/ems_refresh_worker_ansible_tower_configuration%').each do |s|
+        s.key = s.key.sub(
+          %r{^/workers/worker_base/queue_worker_base/ems_refresh_worker/ems_refresh_worker_ansible_tower_configuration},
+          '/workers/worker_base/queue_worker_base/ems_refresh_worker/ems_refresh_worker_ansible_tower_automation'
```

Comment:
I think you're being specific enough in the where clause. Once you use the full path in the where clause, there is no reason to specify it again because you are already sure that you only have the keys that match the path.